com.obsproject.Studio.Plugin.BackgroundRemoval.yaml

```diff
@@ -0,0 +1,67 @@
+id: com.obsproject.Studio.Plugin.BackgroundRemoval
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/BackgroundRemoval
+cleanup:
+  - /include
+  - /share/opencv4
+  - /lib/cmake
+  - /bin
+modules:
+  # onnxruntime has a bash+python+cmake build script - just download/use pre-compiled library
+  # https://onnxruntime.ai/docs/build/inferencing.html
+  - name: onnxruntime
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/microsoft/onnxruntime/releases/download/v1.14.1/onnxruntime-linux-x64-1.14.1.tgz
+        sha256: 010585f534d822bf02d54c7cfc53b57aa3eb0cf34f8fb7503ee519ed18e25d29
+        x-checker-data:
+            type: rotating-url
+            url: https://github.com/microsoft/onnxruntime/releases/latest
+            pattern: "https://github.com/microsoft/onnxruntime/releases/tag/v([0-9.]+)"
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/include ${FLATPAK_DEST}/lib
+      - mv include/* ${FLATPAK_DEST}/include
+      - mv lib/* ${FLATPAK_DEST}/lib
+  # https://docs.opencv.org/4.x/db/d05/tutorial_config_reference.html
+  - name: opencv
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - "-DBUILD_LIST=core,imgproc"
+    sources:
+      - type: archive
+        url: https://github.com/opencv/opencv/archive/refs/tags/4.7.0.tar.gz
+        sha256: 8df0079cdbe179748a18d44731af62a245a45ebf5085223dc03133954c662973
+        x-checker-data:
+            type: rotating-url
+            url: https://github.com/opencv/opencv/releases/latest
+            pattern: "https://github.com/opencv/opencv/releases/tag/v([0-9.]+)"
```

Comment:
Here I think you can use:

```yaml
type: git
tag-pattern: ^([\d.]+)$
```

(Notice that OpenCV just use tag numbers, e.g. `1.2.3`, it doesn't prefix the tags with `v` like `v1.2.3`)